connectors/http/http-base/src/main/java/io/camunda/connector/http/base/services/HttpInteractionService.java

```diff
@@ -42,13 +42,13 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class HTTPService {
+public class HttpInteractionService {
```

Comment:
Why not `HttpService`?

Reply:
Ok, as a reader the whole code is a little bit tough to follow and not easy to understand why there is an `HttpService` and an `HttpInteractionService` to begin with. Its ok to move forward now but there is a need for refactoring as we create new instances of these services for each call like this:
```
 private HttpCommonResult executeRequestDirectly(HttpCommonRequest request)
      throws IOException, InstantiationException, IllegalAccessException {
    ...
    HttpInteractionService httpInteractionService = new HttpInteractionService(gson);
    AuthenticationService authService = new AuthenticationService(gson, requestFactory);
...
}
```

which indicates these are rather utilities and not services.